scripts/perfmon

```diff
@@ -345,11 +345,23 @@ class RRDUpdates:
 
 
 # Consolidation functions:
-supported_consolidation_functions = [ 'sum', 'average', 'max', 'get_percent_fs_usage', 'get_percent_mem_usage' ]
+supported_consolidation_functions = [ 'sum', 'average', 'max', 'get_percent_fs_usage', 'get_percent_log_fs_usage', 'get_percent_mem_usage' ]
 
 def average(mylist):
     return sum(mylist)/float(len(mylist))
 
+def get_percent_log_fs_usage(ignored):
+    "Get the percent usage of the host filesystem for logs partition. Input list is ignored and should be empty"
+    output = commands.getoutput('df /dev/sda5')
+    if not (re.search(r'No such file or directory', output)):
+        output = ' '.join(output.splitlines()[1:])
+        if (output.split()[5]) == '/var/log':
```

Comment:
If this comparison is false, `percentage` will not be set at all, so line 363 will thrown an exception when it attempts `return float(percentage[0:-1])/100.0` with undefined `percentage`.